server/controllers/finance/reports/cashflow/index.js

```diff
@@ -376,7 +378,7 @@ function report(req, res, next) {
     });
     return sourceTotalByTextKeys;
   }
-
+  // eslint-disable-next-line no-shadow
```

Comment:
Same remark as for `aggregateTotalByTextKeys`, try to reuse it by removing this definition of the function.